app/src/main/java/br/ufg/inf/pes/healthhelp/model/Sessao.java

```diff
@@ -15,15 +15,23 @@ public void setUsuario(Usuario usuario) {
     public Sessao() {}
```

Comment:
O construtor dessa classe deve ser privado, pois ela implementa o padrão de projeto Singleton. Se o construtor for público, outras classes poderão criar instâncias de Sessão, o que não pode acontecer.